test/TradeTrustERC721.js

```diff
@@ -65,4 +69,84 @@ contract("TradeTrustErc721", accounts => {
       /VM Exception while processing transaction: revert ERC721: transfer caller is not owner nor approved/
     );
   });
+
+  describe("Surrendered TradeTrustERC721 Work Flow", () => {
+    let tokenRegistryInstanceWithShippingLineWallet;
+    let tokenRegistryAddress;
+    let escrowInstance;
+    let escrowInstanceAddress;
+    let escrowInstance1;
+    let escrowInstanceAddress1;
+
+    beforeEach("", async () => {
+      // Setup in accordance to workflow, starting test after the point of surrendering ERC721 Token
+      tokenRegistryInstanceWithShippingLineWallet = await Erc721.new("foo", "bar", { from: shippingLine });
+      tokenRegistryAddress = tokenRegistryInstanceWithShippingLineWallet.address;
+      escrowInstance = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress = escrowInstance.address;
+      escrowInstance1 = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress1 = escrowInstance1.address;
+      await tokenRegistryInstanceWithShippingLineWallet.safeMint(escrowInstanceAddress, merkleRoot);
+      await escrowInstance.transferTo(tokenRegistryAddress, {
+        from: beneficiary1,
+      });
```

Comment:
Cant we simply mint the token to itself? If the problem is that you cannot safeMint to itself, you will want to have a method to respond to safeMint, if not later all the escrow contract cannot safeTransfer back to the contract also.

Reply:
I believe we can safeMint to itself but the point was to accurately following the surrender flow as mentioned above